### PR TITLE
Add post about Amsterdam genomics workshop

### PR DESCRIPTION
**Pull request by mkuzak**

### CODE REVIEW

**Comment by tracykteal** [comment]
Thanks! Do you want a different title than the post we just posted?

**Comment by mkuzak** [comment]
my mistake, corrected now:)

**Comment by mkuzak** [comment]
@tracykteal I could not get it rendered locally, can you check if it renders properly?

**Comment by tracykteal** [comment]
I think it's not the first in Europe. @lexnederbragt and I taught one last year in Oslo. https://uio-carpentry.github.io/2016-03-15-Oslo-data-bio/ It's the second though! Could just change the language to say 'one of the first'

**Comment by mkuzak** [comment]
Thank you for pointing this out. I did not realise. It does not have to be first :), I'll correct it.

File: _posts/2017-05-04-amsterdam-genomics.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+The event closed with drinks and snacks at nearby Maslov Cafe.
+
+![Post workshop recovery session]({{ site.urlimg }}/blog/2017-ams/maslov.jpeg)
+
+We are just at the beginning of the journey. More work needs to be
+done on improving genomics lesson, and Data Carpentry is looking
+forward to contributions from everyone interested in providing help.
+You can read more about it in the recent [Data Carpentry blog post]
+(http://www.datacarpentry.org/blog/genomics-lessons/)
```
Comment (lexnederbragt) [inline _posts/2017-05-04-amsterdam-genomics.md]: This link does not render on the webpage, maybe due to a space in between the brackets or so...

Comment (mkuzak) [inline _posts/2017-05-04-amsterdam-genomics.md]: thanks @lexnederbragt yes, that's the splitting :) and yes it's late :(

Comment (mkuzak) [inline _posts/2017-05-04-amsterdam-genomics.md]: fixed

**Comment by mkuzak** [comment]
thanks :), I meant the post is late not finding the error

**Pull request merged**